### ISSUE TITLE
could you add python version flag in class `Connection`

### ISSUE THREAD

**Issue by LaoQi** [alfredodeza/remoto#20]
i do some job for ceph-deloy to support archlinux, it seems to work fine.
but the default python version is 3.x in archlinux, so i have to write some code ugly. maybe you can add a param to switch it.
i just change as follow

```
Connection.py
function: _make_connection_string(self, hostname, _needs_ssh=None, use_sudo=None)
line 57: interpreter = 'sudo python' if use_sudo else 'python'
--->interpreter = 'sudo python2' if use_sudo else 'python2'
line 59: interpreter = 'sudo python' if self.sudo else 'python'
--->interpreter = 'sudo python2' if self.sudo else 'python2'
```

i hope you can understand my worse english : -）

**Comment by alfredodeza** [comment]
If I understand this correctly you would like to enforce `python2

**Comment by LaoQi** [comment]
yes , that`s what i meant.

**Comment by alfredodeza** [comment]
Fixed with 9be63f54e3f7760111956244ceb6e67ec2f1d8eb
